include/eve/traits/as_integer.hpp

```diff
@@ -9,15 +9,26 @@
 
 #include <eve/forward.hpp>
 #include <eve/detail/meta.hpp>
+#include <eve/traits/element_type.hpp>
 
 namespace eve
 {
   namespace detail
   {
-    template<typename T> struct sign_of<logical<T>> : sign_of<T> {};
+    template<typename T>
```

Comment:
if it works here, why not put it back in meta.hpp as sign_of<T> ?

Reply:
I can't use `element_type_t` in `meta` - it depends on `forward` which depends on `meta`.
Also `meta` does not know about `eve` types - this is why you had the hack here for logicals.

Reply:
OK good